scripts/globals/items/fishermans_belt.lua

```diff
@@ -14,8 +14,8 @@ local item_object = {}
 
 item_object.onItemCheck = function(target)
     local result = 0
-    if (target:hasStatusEffect(xi.effect.FISHING_IMAGERY) == true) then
-        result = 242
+    if target:hasStatusEffect(xi.effect.FISHING_IMAGERY) == true then
+        result = 235
     end
```

Comment:
Probably would be cleaner to use `xi.effect.FISHING_IMAGERY` instead of the numeric value